filebeat/module/postgresql/_meta/kibana/default/dashboard/Filebeat-Postgresql-overview.json

```diff
@@ -6,13 +6,13 @@
         "kibanaSavedObjectMeta": {
           "searchSourceJSON": "{\"filter\":[]}"
         },
-        "savedSearchId": "ef9da9b0-776c-11e7-9e03-393c0d1aa325",
+        "savedSearchId": "PostgreSQL All Logs",
```

Comment:
Index pattern change was not enough. It was necessary to change the ids otherwise searches and visualizations couldn't be located. Also, I checked the working dashboards and those JSONs contain similar IDs. I wanted to be consistent with the existing correct dashboard JSONs.